addons/folo/common/src/main/java/org/commonjava/indy/folo/data/FoloRecordCassandra.java

```diff
@@ -92,7 +92,7 @@ private static String createFoloRecordsTable( String keyspace )
     private static String createFoloKeyspace( String keyspace)
     {
         return "CREATE KEYSPACE IF NOT EXISTS " + keyspace
-                + " WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor':3 };";
+                + " WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor':1 };";
```

Comment:
We may need to make the replication_factor configurable. As John mentioned, we can declare one for all the noncontent stuff. So we can make a pr for this type of thing after merging.

Reply:
+1